python/py-lmfit/Portfile

```diff
@@ -1,45 +1,50 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem              1.0
-PortGroup               python 1.0
-
-name                    py-lmfit
-version                 0.9.13
-revision                0
-categories-append       math
-license                 BSD
-maintainers             {gmail.com:jjstickel @jjstickel} {reneeotten @reneeotten} openmaintainer
-description             Least-Squares Minimization with Bounds and Constraints
-long_description        Built on top of scipy.optimize, lmfit provides a\
-                        Parameter object which can be set as fixed or free,\
-                        can have upper and/or lower bounds, or can be written\
-                        in terms of algebraic constraints of other Parameters.
-platforms               darwin
-homepage                https://lmfit.github.io/lmfit-py/
-master_sites            pypi:l/lmfit/
-distname                lmfit-${version}
-
-checksums               rmd160  e9c725baacf92b9255960e76fd0ed32ba52dd92f \
-                        sha256  aaabeb5276cebe7d7c4172c6b1ada5df95fd020e5a43070543961c60bd6f0ec1 \
-                        size    1457657
-
-python.versions         27 35 36 37
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-lmfit
+version             0.9.14
+revision            0
+
+categories-append   math
+platforms           darwin
+supported_archs     noarch
```

Comment:
Did you mean to add `supported_archs     noarch`?

That means that the `depends_lib`s below should probably be changed to `depends_run` right?

Reply:
Yes, that addition was intentional - the port does not install any architecture-specific files. I don’t see how that is related to lib/run dependencies. As far as I can tell this is fine and what’s done for many other ports.

Reply:
Then those other ports are incorrect. A library dependency is required to build the port. Since nothing gets built, the dependencies are thus only needed at runtime for it to work in this case.

Reply:
In this case it can probably become a run dependency only as there are no checks for the package being present during the build. But I don’t think it can be generalized as “if there are no architecture-dependent files installed, you cannot have lib-dependencies”. At least that’s what you seem to suggest, or?

It’s not always easy to figure out from the “install_requires” field if it’s needed as lib/run dependency, and setting it to a lib dependency will always work. Anyway, this distinction is at the most only relevant for the buildbot; for the end user he/she will need the packages anyway and the order in which they get installed then really doesn’t matter. I am inclined to merge this as-is and perhaps go through “my” ports at a later point to see whether it is needed to change from lib/run dependencies...

Reply:
Sounds good.